test/integration/noderesourcetopology_test.go

```diff
@@ -1035,7 +1331,7 @@ func checkPodSchedulingFailed(c clientset.Interface, podNamespace, podName strin
 	}
 	events, err := c.CoreV1().Events(podNamespace).List(context.TODO(), opts)
 	if err != nil {
-		klog.ErrorS(err, "cannot get events for pod %s/%s", podNamespace, podName)
+		klog.ErrorS(err, "cannot get events for pod", podNamespace, podName)
```

Comment:
thanks for this fix! since we are at it, we may want to fully utilize the structured format:
```
klog.ErrorS(err, "cannot get events for pod", "namespace", podNamespace, "name", podName)
```